test/helpers/expectEvent.js

```diff
@@ -35,7 +44,17 @@ function isBigNumber (object) {
     (object.constructor && object.constructor.name === 'BigNumber');
 }
 
+function decodeLogs (logs, events) {
+  return _.flatten(logs.map(log =>
```

Comment:
It's really not necessary to use `_.flatten` here because there will always be only one level of nesting. I think the following works:
```suggestion
  return Array.prototype.concat(...logs.map(log =>
```

Reply:
That does work, yes! My JS kung fu isn't that strong to come up with that on my own. We can get rid of `lodash` then, and you'll sleep better :)